### DISCUSSION TITLE
User context caching documentation

### DISCUSSION THREAD

**Discussion by RafaelKr** [sulu/SuluCommunityBundle#176] [Q&A]
I read the Sulu documentation about [User context caching](https://docs.sulu.io/en/2.6/cookbook/user-context-caching.html). It also mentions the SuluCommunityBundle. So I assume it's also required to configure User context caching, but it's not mentioned in the documentation of SuluCommunityBundle.

But it's required, right? Maybe then there could be also a reference in the SuluCommunityBundle.

**Comment by alexander-schranz** [comment]
@RafaelKr it is not required if you don't require user context based caching. Example if the only thing of the community bundle usages is provide registration, profile, password forget for static private symfony routes there is no requirement for user context based caching. So there are lot of sites I built in the past which didn't have the need for user context based caching but had the Community Bundle in it for the application part of that project, but content was never behind the firewall in that projects.

So User context based caching is required if you put Content Pages, Articles, ... behind the firewall.

**Reply by RafaelKr** [reply]
@alexander-schranz Thanks for your answer!

If I understand you correctly it's only related to routes (pages, articles, etc.) coming from the Sulu CMS. When we add own routes behind the firewall they're not affected by the SuluHttpCache anyways, right?

Also we're using the embed template described at https://github.com/sulu/SuluCommunityBundle/blob/2.0/Resources/doc/4-login.md#embed_template. Is this also not affected by the SuluHttpCache as it's using ESI?
